processes/processes.go

```diff
@@ -23,85 +23,160 @@ import (
 	"fmt"
 	"os"
 	"strconv"
+	"strings"
 	"time"
 
-	"github.com/intelsdi-x/snap/control/plugin"
-	"github.com/intelsdi-x/snap/control/plugin/cpolicy"
-	"github.com/intelsdi-x/snap/core"
-	"github.com/intelsdi-x/snap/core/serror"
-
-	"github.com/intelsdi-x/snap-plugin-utilities/config"
-	"github.com/intelsdi-x/snap-plugin-utilities/str"
+	"github.com/intelsdi-x/snap-plugin-lib-go/v1/plugin"
 )
 
 const (
-	//plugin name
-	pluginName = "processes"
+	// PluginName is name of plugin
+	PluginName = "processes"
+	// PluginVersion is version of plugin
+	PluginVersion = 8
+
 	//plugin vendor
 	pluginVendor = "intel"
 	// fs is proc filesystem
 	fs = "procfs"
-	//version of plugin
-	version = 7
+
+	// Namespace offsets
+	nsCategory  = 3
+	nsProcName  = 4
+	nsStateName = 4
+	nsPid       = 5
+	nsPsCount   = 5
+	nsPidMetric = 6
```

Comment:
Could you provide more info in the comment to make managing of this code easier in the future.